### PR TITLE
make postprocessing more flexible

### LINKED ISSUE #351: Make drak-postprocess unit file templated
Allow drak-postprocess to be scaled more easily

### PR DESCRIPTION
**Pull request by BonusPlay**
close #351

### CODE REVIEW

File: drakcore/drakcore/systemd/drak-postprocess@.service [hunk 1]
```diff
@@ -11,8 +11,9 @@ Restart=on-failure
 RestartSec=5
 StartLimitInterval=60s
 StartLimitBurst=3
-KillSignal=SIGQUIT
 WorkingDirectory=/var/lib/drakcore
+KillMode=process
+TimeoutStopSec=700
```
Comment (icedevml) [inline drakcore/drakcore/systemd/drak-postprocess@.service:16]: I think this was copied from `drakrun@` but I think postprocess is not supposed to run so long
```suggestion
TimeoutStopSec=120
```

**Review by icedevml — APPROVED** [review]

File: drakcore/drakcore/systemd/drak-postprocess@.service (hunk 1 above)
Comment (chivay) [reply drakcore/drakcore/systemd/drak-postprocess@.service:16]: I've looked at some docs about KillMode and found that:
>Note that it is not recommended to set KillMode= to process or even none

So, I guess that we shouldn't use it anywhere in the codebase.

Comment (icedevml) [reply drakcore/drakcore/systemd/drak-postprocess@.service:16]: I remember this was set on purpose, due to the fact that some service was not able to stop using the default mechanism. I don't remember the exact reason :(

**Comment by BonusPlay** [comment]
This needs to be noted as a breaking change in release notes!

**Pull request merged**